metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -141,6 +145,9 @@ object InitializationOptions {
       isHttpEnabled = jsonObj.getBooleanOption("isHttpEnabled"),
       isCommandInHtmlSupported =
         jsonObj.getBooleanOption("isCommandInHtmlSupported"),
+      commandInHtmlFormat = jsonObj
+        .getStringOption("commandInHtmlFormat")
+        .flatMap(CommandHTMLFormat.fromString),
```

Comment:
We should deprecate `isCommandInHtmlSupported` in favor of `commandInHtmlFormat`. 
Since you removed `isCommandInHtmlSupported` from `if`'s I wonder what we should do in other editors. For sure, vscode has to be reflect those changes and set `isCommandInHtmlSupported` instead of `isCommandInHtmlSupported`. 
@ckipp01 what about nvim?

If `isCommandInHtmlSupported` was set in vscode then maybe we should remove it right away? I can do this on vscode side, but I don't know if it won't break some other editor.

Reply:
`isCommandInHtmlSupported`'s value in initialization options completely ignored. Setting it has no effect.
I think we should simply removed it.